src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -204,7 +205,16 @@ public function caution($message)
      */
     public function table(array $headers, array $rows)
     {
-        $headers = array_map(function ($value) { return sprintf('<info>%s</>', $value); }, $headers);
+        array_walk_recursive($headers, function (&$value) {
```

Comment:
not really recusive, but it can be one/two-dimensional array which is already covered by [test](https://github.com/aitboudad/symfony/blob/issue_18573/src/Symfony/Component/Console/Tests/Fixtures/Style/SymfonyStyle/command/command_8.php#L10-L13)